README.md

```diff
@@ -1,8 +1,119 @@
-# react-front
+# cnaas-front
+
+## simple standalone dev setup
 
 1. download node
 2. download npm
-3. navigate to the react-front folder
+3. navigate to the cnaas-front folder
 4. run `npm i` to install all the dependencies
 5. run: `npm start` to bundle the js files
 6. navigate to `http://localhost:1234` in browser
+
+
+## full interactive dev setup with docker-compose
+
+This guide will show you how to set up cnaas-nms on your local machine for development.
+The components will run in Docker-containers.
+
+The cnaas-nms frontend will also run inside a Docker container and update interactively with
+every code change.
+
+
+### clone the repositories
+
+Clone the source code for frontend and API. Both should be in the same parent directory.
+
+    git clone https://github.com/SUNET/cnaas-front.git cnaas-front
+    git clone https://github.com/SUNET/cnaas-nms.git cnaas-nms
+
+
+### setup and run docker-compose
+
+Navigate to `cnaas-front/`.
+
+    cd cnaas-front
+
+You will find a `docker-compose.yaml` file here.
+
+This file is used to run docker-compose and build all containers. This takes forever the first time.
+
+    docker-compose up -d
+
+As soon as the built is finished, docker-compose will start the containers.
+
+
+### set up the auth container
+
+A script will set up the auth container for you.
+This script will create a certificate to verify JWTokens issued by the auth server.
+It will also create a user named "cnaas" with password "cnaascnaascnaas".
```

Comment:
is this password correct? Didn't work for me but I could create another user that worked

Reply:
The password worked for me when I just tested it - however, the auth server gave me an invalid token. It's content part translated to this:

```json
{"iat":1632994051,"nbf":1632994051,"jti":"dfe4f74d-4203-47b9-944a-1aed4b68045e","sub":"cnaas","fresh":false,"type":"access"
```

I.e., this JSON blob is missing the last `}` and is therefore unparseable by CNaaS-NMS

Reply:
@indy-independence Password seems correct and has worked multiple times so far for us.

@lunkwill42 I also encountered this issue when trying to decode the token with base64. It seems to happen for all cnaas-nms tokens for me, so I wonder if this is related to this PR at all.

I suspect your issues being related to the missing `killall` command -- if you don't kill all currently running uwsgi processes, the newly copyed JWT-token will not take effect.
Maybe there are also issues with file system permissions after copying the tokens as root.
I will look into that.